src/features/Chat/ChatWindow.tsx

```diff
@@ -3,18 +3,98 @@ import { useNavigate } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 
 import { palette } from '@/components/variables';
+import { Profile as ProfileIcon } from '@/components/Icons/Profile';
 import Text from '@/components/Text';
-import { TextButton } from '@/components/Buttons';
+import TextInput from '@/components/TextInput';
+import { Button, IconButton, TextButton } from '@/components/Buttons';
+import { useState } from 'react';
+
+const searchInputIconSize = 24;
+const closeInputIconSize = 34;
 
 const ChatWindow = () => {
   const navigate = useNavigate();
   const { t } = useTranslation('chat');
-  const chats = [];
+  const chats = [{}];
+  const [showSearch, setShowSearch] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+  const [inputValue, setInputValue] = useState('');
 
   return chats.length ? (
```

Comment:
Hmm I thought I would do it in the next PR. The components need to be structured a bit different because we have the different themes in the Chat Window for active, archived and blocked chats.

Reply:
So this goes well hand in hand with that.

Reply:
Ok, thats fine too,( to do it in next PR)
I would hope we can have one ChatWindow and one ChatMenu. 
The ChatCategory could be passed as a prop, or then selected from the store, whichever works.